app/models/enhancements/artefact.rb

```diff
@@ -8,6 +8,11 @@ class Artefact
   # without persisting it
   attr_accessor :indexable_content
 
+  # Add a non-field attribute so we can pass the redirect_url over to the Router
+  # without persisting it
+  attr_accessor :redirect_url
```

Comment:
There's only this task - https://github.com/alphagov/panopticon/blob/master/lib/tasks/router.rake.  It only covers live artefacts at the moment.  It might be worth extending to cover archived artefacts as well now that we have the option to do that.

Actually, thinking about it, we'd need to import any relevant redirects from router-data before doing that otherwise they'd all get overwritten. Maybe that's a follow-up piece of work. We should still store the redirect_url for now, so that we have the option to do this in future.